packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 48.21.0 - June 17, 2020
+
+## Added
+
+#### docs
+
+- Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
+
+#### salesforcedx-vscode-core
+
+- changes for release ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```

Comment:
I wasn't sure what we wanted to include here, if anything, for the source-deploy-retrieve library.

Reply:
We do, but we need to word it as supporting Aura and LWC for the performance enhancement stuff cc @brpowell @amandastern99